ui/public/config.json

```diff
@@ -1,5 +1,5 @@
   {
-    "SERVER_URL": "http://localhost:5002",
+    "SERVER_URL": "http://localhost:5555",
```

Comment:
Q: Why did the server port change?
We should make sure to document this in release notes

Reply:
This is only the port used for development, should not affect external users.